web_access_rule_buttons/__manifest__.py

```diff
@@ -4,16 +4,18 @@
 {
     "name": "Web Access Rules Buttons",
     "summary": "Disable Edit button if access rules prevent this action",
-    "version": "14.0.1.0.0",
-    "author": "Camptocamp, Onestein, Odoo Community Association (OCA)",
+    "version": "15.0.1.0.0",
+    "author": "Camptocamp, Onestein, Ryan Cole, Odoo Community Association (OCA)",
```

Comment:
Unfortunately a migration does not qualify as author (unless a big refactor takes place).
You can add yourself as a contributor in the `readme/CONTRIBUTORS.rst` file